descriptors/ModuleDescriptor-template.json

```diff
@@ -602,7 +602,7 @@
     },
     {
       "id": "instance-storage",
-      "version": "10.0"
+      "version": "10.0 10.3"
```

Comment:
On the second thought, let's leave 10.3 version only. With these changes we won't be able to work with instance-storage v10.0